packages/core/src/item.ts

```diff
@@ -269,13 +269,21 @@ export enum AuditType {
     WeakPassword = "weak_password",
     ReusedPassword = "reused_password",
     CompromisedPassword = "compromised_password",
+    ExpiredItem = "expired_item",
 }
 
 export interface AuditResult {
     type: AuditType;
     fieldIndex: number;
 }
 
+export enum ExpiryFrequencyOption {
+    Manual = "manual",
+    Trimestrially = "trimestrially",
+    Semestrially = "semestrially",
+    Yearly = "yearly",
+}
+
```

Comment:
Those options seem weird to me. I think more sensible defaults would be `weekly`, `monthly`, `quarterly`, `yearly`. But since `monthly` and `quarterly` and `yearly` can't be accurately expressed in days, it's probably better to use `7 days`, `30 days`, `90 days`, `365 days`.

Reply:
I've never seen rotation policies shorter than 3 months, and even those were for very extreme cases. The most common is yearly, but I'm happy to go with 30, 90, 120, 365 days as options in the UI, instead.